### DISCUSSION TITLE
Wix0200 error when trying to include NetFxExtension on Wix v5

### DISCUSSION THREAD

**Discussion by JosephHerreraDev** [wixtoolset/issues#8218] [Questions] [answered]
I want to verify that .net 8 is installed on the end user system to install my software. 
I installed the nuget package: WixToolset.Netfx
I include the namespace
`xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension"`
and the code from the example from the page
`<netfx:DotNetCompatibilityCheck
  Property="DOTNETRUNTIMECHECK"
  RollForward="major"
  RuntimeType="desktop"
  Platform="x64"
  Version="8.0.0"
/>`
But every time I try to build the installer it displays the following error:
The Package element contains an unhandled extension element 'DotNetCompatibilityCheck'. Please ensure that the extension for elements in the 'http://schemas.microsoft.com/wix/NetFxExtension' namespace has been provided.
What can I do? Thank you for the help

**Comment by Thylon1** [comment, marked as answer]
You cannot place a <netfx:DotNetCompatibilityCheck /> element directly inside a package element, which causes the Wix0200 error.
You can only put it below a Bundle or Fragment element as seen in the [documentation](https://wixtoolset.org/docs/schema/netfx/dotnetcoresearch/).

You should also probably use the namespace `xmlns:netfx="http://wixtoolset.org/schemas/v4/wxs/netfx"` to include netfx instead of `xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension".`
